packages/web3torrent/src/components/torrent-info/TorrentInfo.tsx

```diff
@@ -20,6 +21,8 @@ const TorrentInfo: React.FC<TorrentInfoProps> = ({
   channelCache = {},
   mySigningAddress
 }) => {
+  const closeChannel = useContext(WebTorrentContext).paymentChannelClient.closeChannel;
```

Comment:
I think you can't assign function like this. `this` won't work correctly in `closeChannel`. Try just storing the `paymentChannelClient`.

Reply:
thanks. It has been a pain trying to get storybook to work again!